src/lib/chat/index.ts

```diff
@@ -27,7 +27,8 @@ export class Chat {
       modules: [new GroupChannelModule()],
     }) as SendbirdGroupChat;
 
-    this.sendbird.options.sessionTokenRefreshTimeout = 1800; // Max
+    this.sendbird.options.sessionTokenRefreshTimeout = 1800; // 30min Max
+    this.sendbird.options.websocketResponseTimeout = 1800; // 30min Max
```

Comment:
I'm guessing we probably don't want this.

Reply:
removed